src/DonationForms/DataTransferObjects/DonationFormGoalData.php

```diff
@@ -77,21 +80,15 @@ public function getCurrentAmount()
 
         $query->form($this->formId);
 
-        if($this->goalProgressType->isCustom()) {
-            $query->between($this->goalStartDate, $this->goalEndDate);
-        }
-
         switch ($this->goalType):
             case GoalType::DONORS():
+            case GoalType::DONORS_FROM_SUBSCRIPTIONS():
                 return $query->countDonors();
             case GoalType::DONATIONS():
-                return $query->count();
             case GoalType::SUBSCRIPTIONS():
                 return $query->count();
             case GoalType::AMOUNT_FROM_SUBSCRIPTIONS():
                 return $query->sumInitialAmount();
-            case GoalType::DONORS_FROM_SUBSCRIPTIONS():
-                return $query->countDonors();
```

Comment:
Let's add an `@unreleased` tag for these changes.

Reply:
For what? I just grouped the related cases.